src/DataStructures/Variables.hpp

```diff
@@ -909,7 +909,7 @@ struct Subitems<TagList, Tag,
 
 namespace Tags {
 template <size_t N, typename T>
-struct TempTensor {
+struct TempTensor : db::SimpleTag {
```

Comment:
I have a few buffers that are passed in to the computation functions to avoid allocations, and the computation functions are intended to act on a `DataBox`. Do you have an alternative suggestion for how to resolve that clash?

Reply:
I'll need to think about any possible alternatives a bit. Thank you for explaining the use case, that makes sense :)

Reply:
I can't find where this tag is used inside a DataBox in this PR. I'd be more comfortable with deferring this change to where it is actually necessary so I have more context to think about it. I also don't want this PR to be held up on this, since the rest looks good :)

Reply:
This is used via the type alias `TempScalar` in the tag lists for the quantities that need to be passed into the computation by `not_null` pointer. An example of one of those type lists is on `Equations.hpp:294`. This change is required to actually used those computational items via the `db::mutate_apply` in the tests.

Reply:
@nilsdeppe is this resolved for you?